app/controllers/journal_entries_controller.rb

```diff
@@ -0,0 +1,65 @@
+class JournalEntriesController < ApplicationController
+  before_action :set_journal
+
+  def set_journal
+    if @journal = Journal.find(params[:journal_id])
+    else ActiveRecord::RecordNotFound
+      flash[:error] = t('entry.message.error.show')
+    end
+  end
+
+  def index
+    @journal_entries = JournalEntry.all
+    redirect_to journal_path(@journal)
+  end
+  
+  def new
+    @journal_entry = @journal.journal_entries.new
+  end
+
+  def create
+    @journal = Journal.find(params[:journal_id])
+    @journal_entry = @journal.journal_entries.new(journal_entry_params)
+
+    if @journal_entry.save
+      flash[:success] = t('entry.message.success.create')
+      redirect_to journal_path(@journal)
+    else
+      flash[:error] = t('entry.message.error.create')
+      render :new
+    end
+  end
+
+  def destroy
+    @journal_entry = @journal.journal_entries.find(params[:id])
+    if @journal_entry.destroy
+      flash[:success] = t('entry.message.success.delete')
+      redirect_to journal_path(@journal)
+    else
+      flash[:error] = t('entry.message.error.delete')
+      redirect_to journal_path(@journal)
+    end
+  end
+
+  def show
+    @journal_entry = @journal.journal_entries.find(params[:id])
+  end
+
+  def update
+    @journal_entry = @journal.journal_entries.find(params[:id])
+    if @journal_entry.update(journal_entry_params)
+      flash[:success] = t('entry.message.success.update')
+      redirect_to journal_path(@journal)
+    else
+      flash[:error] = t('entry.message.error.update')
+      redirect_to journal_path(@journal) 
+    end
+  end
+
+  private
+  
+  def journal_entry_params
+    params.require(:journal_entry).permit(:title, 
+                                          :content)
+  end
```

Comment:
You can create a method like
def find_journal_entries
  @journal_entry = @journal.journal_entries.find(params[:id])
end

and use a before_action for this like

before_action :find_journal_entries, only: %i(show destroy)